build.gradle

```diff
@@ -38,12 +38,20 @@ repositories {
 dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
 
-    def tapestry_version='5.3.7'
+    def tapestry_version='5.5.0-alpha-1'
```

Comment:
@kaosko Is the code going to be `merged` into the tapestry5 core or any other existing module or is it going to be available as a standalone module?